### PR TITLE
Update data_tutorial.md

### PR DESCRIPTION
**Pull request by eknesterova**

### CODE REVIEW

**Comment by eknesterova** [comment]
@ChirkinaTanya @azhilina  please review

**Review by ChirkinaTanya — APPROVED** [review]

File: data_tutorial.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This tutorial describes the steps to add symbols to the TradingView chart at onc
 >__Important__
 >
 > 1. Symbols are uploaded into the TradingView storage every 10 minutes.
```
Comment (anachuprina) [inline data_tutorial.md:30]: Is it still accurate that the storage is updated every 10 minutes?

File: data_tutorial.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ This tutorial describes the steps to add symbols to the TradingView chart at onc
...
-> Hence, the maximum time for symbols to appear on the chart is 10 minutes.
+> Hence, the time of appearance of symbols on the chart is about 15 minutes.
```
Comment (anachuprina) [inline data_tutorial.md:31]:
```suggestion
> Hence, symbols appear on the chart within 15 minutes.
```

File: data_tutorial.md (hunk 1 above)
Comment (eknesterova) [reply data_tutorial.md:30]: Yes, that information is correct

**Pull request merged**